### PR TITLE
Update data for Philippines

### PR DESCRIPTION
**Pull request by xomanuel**
## Description

1. Update data sources of the Philippines as of September.

Note: The update includes only on-grid power plants, Philippines DOE opt to update the off-grid power plant in annual basis

2. Added resources in the PH parser

3. Added zone capacities

### CODE REVIEW

**Review by VIKTORVAV99 — COMMENTED** [review]
The capacity changes look good but I'll let @pierresegonne have the final say on the power plant mappings in the parser.

File: config/zones/PH-MI.yaml [hunk 1]
```diff
@@ -3,13 +3,22 @@ bounding_box:
     - 4.57480853775256
   - - 127.08266582665826
     - 10.957111072508837
+capacity:
+  battery storage: 156
```
Comment (mathilde-daugy) [inline config/zones/PH-MI.yaml:7]: I think battery storage capacity here is 49 MW. 156 MW is the total for PH :)

**Comment by mathilde-daugy** [comment]
@xomanuel, could you point me to where you found the information for the additional mapped plants ? Thanks :)

**Comment by mathilde-daugy** [comment]
We also need to update the parser to collect battery storage points. From a quick glance it seems like the parser can only handle hydro storage for now

**Comment by xomanuel** [comment]
> @xomanuel, could you point me to where you found the information for the additional mapped plants ? Thanks :)

Hi, \i gathered the additional plants at IEMOP website under [Updates on the Market Network Model](https://www.iemop.ph/market-reports/updates-on-the-market-network-model/)



> We also need to update the parser to collect battery storage points. From a quick glance it seems like the parser can only handle hydro storage for now

Noted on this. Hope to have a parser to collect battery storage.

Thank you so much!

**Comment by mathilde-daugy** [comment]
@xomanuel are you able to update the parser to collect battery storage info? if not we can take care of it in a seperate PR :)

**Comment by xomanuel** [comment]
@mathilde-daugy I think I let the experts on how to do the parser for the battery storage in the next PR 😊

**Comment by mathilde-daugy** [comment]
ok, i will approve for now and take care of updating the parser

**Review by mathilde-daugy — APPROVED** [review]

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM! 🎉

**Pull request merged**